src/lib/components/Authenticate/index.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+  import Login from './Login.svelte';
+  import SignUp from './SignUp.svelte';
+
+  let login = false;
+  function handleChangeLogin() {
+    if (login == false) {
+      login = true;
+    } else if (login == true) {
+      login = false;
+    }
+  }
+</script>
+
+<div>
+  {#if login}
+    <Login on:change={(e) => console.log(e.detail)} on:click={() => handleChangeLogin()} />
+  {/if}
+
+  {#if !login}
+    <SignUp on:click={() => handleChangeLogin()} />
+  {/if}
```

Comment:
As we are using a boolean, there's only 2 possible values here. Theres no need for a second `if`.

```suggestion
  {#if isLogin}
    <Login on:toggleForm={handleToggleForm} />
  {:else}
    <SignUp on:toggleForm={handleToggleForm} />
  {/if}
```